code/modules/mob/new_player/sprite_accessories/hair.dm

```diff
@@ -716,3 +716,11 @@
 /datum/sprite_accessory/hair/oxton
 	name = "Oxton"
 	icon_state = "hair_oxton"
+
+/datum/sprite_accessory/hair/row_braid
+    name = "Row Braid"
+	icon_state = "hair_row_braid_s"
+
+/datum/sprite_accessory/hair/croft
+    name = "Croft"
+	icon_state ="hair_croft_s"
```

Comment:
```suggestion
/datum/sprite_accessory/hair/row_braid
	name = "Row Braid"
	icon_state = "hair_row_braid_s"

/datum/sprite_accessory/hair/croft
	name = "Croft"
	icon_state ="hair_croft_s"
```

you can't use spaces to indent with, it makes it inconsistent with the tabs below + we don't allow space indentation regardless